### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by vlad-solomon** [epicweb-dev/restore-scroll#10]
Not even having a demo present in the README or as a link in the project settings tell you everything you need to know about this package

**Comment by vlad-solomon** [comment]
This is my code:



```jsx
// Home.jsx
export default function Home() {
    const { data, isLoading } = useData();
    const [columns, setColumns] = useState(3);

    if (isLoading) {
        return "loading...";
    }

    return (
        <div className="flex gap-8 max-w-[800px] m-auto h-[100dvh]">
            {Array.from({ length: columns }, (_, index) => index).map(
                (column) => (
                    <>
                        <Column key={column} id={`column-${column}`}>
                            {data
                                .filter(
                                    (_, index) => index % columns === column
                                )
                                .map((photo) => (
                                    <Frame
                                        key={photo.slug.current}
                                        slug={photo.slug.current}
                                        src={urlFor(photo.image.asset._ref)}
                                    />
                                ))}
                        </Column>
                        <ElementScrollRestoration
                            elementQuery={`#column-${column}`}
                        />
                    </>
                )
            )}
        </div>
    );
}
```

```jsx
// Column.jsx
export default function Column({ children, id }) {
    return (
        <div className="w-full flex-1 overflow-auto" id={id}>
            {children}
        </div>
    );
}
```

**Comment by kentcdodds** [comment]
I've created a demo and added it to the README. I'm afraid I don't have the bandwidth to troubleshoot other people's projects. Sorry.

**Comment by vlad-solomon** [comment]
Seems you had just enough bandwidth to create a demo one year after releasing a half-baked package. No need to troubleshoot my project, I showed you that I respected the instructions in the README and your approach just doesn't work. Why? I don't really know. I wasn't looking for you to solve my project or my problem, just wanted to let you know that in my case this approach is not working.

There's no need for you to be sorry, just wanted to give you a heads up if you would ever like to develop this project more. If not, archiving it might be a better solution.
